db/migrate/20160224175308_create_tools.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTools < ActiveRecord::Migration
+  def change
+    create_table :tools do |t|
+      t.string :title
+      t.text :description
+      t.integer :rents, default: 0
```

Comment:
:null => false.

Also I'm not sure how this works. We don't track renting except incrementing a number? Don't we want to know who rented it and when? We could still have this field as a cache value if necessary, but it seems we would want to know who rented a tool.